Assets/Scripts/MultiBallPwrController.cs

```diff
@@ -19,7 +19,10 @@ public class MultiBallPwrController : PowerupController
         currentBallPosition = currentBall.transform.position;
         currentBallVelocity = currentBall.GetComponent<Rigidbody>().velocity;
         for(int i = 0; i < (ballsToMake - ballCount); i++){
-            Vector3 newBallVelocity = new Vector3((currentBallVelocity.x + angleVariation * (i + 1)), currentBallVelocity.y, currentBallVelocity.z);
+            // Vector3 newBallVelocity = new Vector3((currentBallVelocity.x + angleVariation * (i + 1)), currentBallVelocity.y, currentBallVelocity.z);
+            Vector3 newBallVelocity = Quaternion.AngleAxis(angleVariation * (i + 1), new Vector3(0, 1, 0)) * currentBallVelocity;
```

Comment:
Is this the "jitter" approach we talked about?

Reply:
No, that's still the last update prior to that conversation. Hadn't made it to master yet. This rotates around the y-axis rather than just increasing the magnitude in x.